applications/ISO20022-Implementation-POC.md

```diff
@@ -0,0 +1,178 @@
+- Allow for upgrading registrations and unregistering 
+    -   https://github.com/encointer/pallets/pull/254
+- Allow reputables to endorse
+    - https://github.com/encointer/pallets/pull/268
+- Geohash location validation algorithm
+    - https://github.com/encointer/pallets/pull/55
+- Constant time random permutation for meetup assignment
+    -  https://github.com/encointer/pallets/pull/68
+
+#### Misc:
+- Implement custom RPCs
+    - https://github.com/encointer/pallets/pull/196
+- Implement storage migrations for Weight V2
+    - https://github.com/encointer/pallets/pull/308
+- Various polkadot/substrate dependency bumps
+    - https://github.com/encointer/pallets/pull/299
+    - https://github.com/encointer/encointer-parachain/pull/171
+- Fee payment in Encointer community currency
+    - https://github.com/encointer/encointer-node/pull/194/files
+    - https://github.com/encointer/encointer-node/pull/226
+    - https://github.com/encointer/pallets/pull/190
+- Integrate try runtime 
+    - https://github.com/encointer/encointer-parachain/pull/177/files
+    - https://github.com/encointer/encointer-node/pull/312/files
+
+	
+#### Non rust contributions:
+- Build an indexer using qubquery
+    - https://github.com/encointer/subquery (Main author)
+- Build an accounting tool for encointer businesses
+    - https://github.com/encointer/accounting-frontend (Main author)
+    - https://github.com/SolidbitGmbH/encointer-accounting-backend (Main author)
+- Build an integration for Encointer personhood in KILT
+    - https://github.com/encointer/kilt-personhood (Main author)
+- Contributions to encointer js
+    - https://github.com/encointer/encointer-js/pull/78
+
+
+Piero has previously received a W3F grant for a different project(see above).
+
+
+### Team Code Repos
+
+- https://github.com/pifragile
+- https://github.com/SolidbitGmbH
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- https://github.com/pifragile
+
+
+### Team LinkedIn Profiles
+
+- https://www.linkedin.com/in/piero-guicciardi-66b939152/
+
+
+## Development Status :open_book:
+This project has not started yet, but there has been some previous related work in the ecosystem (see references), which we studied and will build upon.
+
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Estimated duration:** 1 month
+- **FTE:**  0.25 (5 person days)
+- **Total Costs:** 5200 CHF (5850 USD)
+
+
+
+### Milestone 1 — Ecosystem Research and Technical Design
+
+- **Estimated duration:** 1 month
+- **FTE:**  0.25 (5 person days)
+- **Costs:** 5200 CHF (5850 USD)
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0e. | Article | We will publish an **article** that outlines the current state of other blockchain's ISO20022 development as well as technical designs how to solve this for Polkadot.|
```

Comment:
Thanks for the update, @pifragile. Happy to move forward. Just to give the article a bit of structure, I would like to ask that a bibliography and description of how the material was sourced be included. Can be part of the article.

```suggestion
| -----: | ----------- | ------------- |
| **0c.** | Methodology | Description of how the information was sourced. |
| **0e.** | Bibliography | List of sources |
| 1. | Article | We will publish an **article** that outlines the current state of other blockchain's ISO20022 development as well as technical designs how to solve this for Polkadot.|
```

Reply:
sure, makes sense!